[Integrations] Testcontainers, Kubernetes Operator, Kratix (#4146)
dapr/docs: daprdocs/content/en/developing-applications/integrations/kubernetes-operator.md

---
type: docs
title: "How to: Use the Dapr Kubernetes Operator"
linkTitle: "Dapr Kubernetes Operator"
weight: 7000

> description: "Use the Dapr Kubernetes Operator to manage the Dapr lifecycle"

Reviewer: ```suggestion
description: "Use the Dapr Kubernetes Operator to manage the Dapr control plane"
```

---

> The Dapr Kubernetes Operator manages the lifecycle for Dapr and its components. Use the operator to automate the tasks required when oeprating Dapr in Kubernetes mode. 

Reviewer: ```suggestion
The Dapr Kubernetes Operator manages the lifecycle for Dapr and its components. Use the operator to automate the tasks required when operating Dapr in Kubernetes mode. 
```

> The Dapr Kubernetes Operator manages the lifecycle for Dapr and its components. Use the operator to automate the tasks required when operating Dapr in Kubernetes mode. 

Reviewer: ```suggestion
You can use the Dapr Kubernetes Operator to manage the Dapr control plane. Use the operator to automate the tasks required to manage the lifecycle of Dapr control plane in Kubernetes mode. 
```

---
type: docs
title: "How to: Use the Dapr Kubernetes Operator"
linkTitle: "Dapr Kubernetes Operator"
weight: 7000
description: "Use the Dapr Kubernetes Operator to manage the Dapr lifecycle"
---

The Dapr Kubernetes Operator manages the lifecycle for Dapr and its components. Use the operator to automate the tasks required when oeprating Dapr in Kubernetes mode. 

{{% alert title="Note" color="primary" %}}

> The Dapr Kubernetes Operator is currently a Dapr sandbox project.

Reviewer: You can remove this comment. It should be in dapr org as per https://github.com/dapr/community/issues/425

{{% /alert %}}
 

> {{< button text="Install and use the Dapr Kubernetes Operator" link="https://github.com/dapr-sandbox/dapr-kubernetes-operator" >}}

Reviewer: ```suggestion
{{< button text="Install and use the Dapr Kubernetes Operator" link="https://github.com/dapr/dapr-kubernetes-operator" >}}
```
